src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,87 @@
-import React from 'react';
+import React, { FC, useCallback, useState } from 'react';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onAdd: (newMovie: Movie) => void;
+}
+
+const baseImgUrl = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+export const FindMovie: FC<Props> = ({
+  onAdd,
+}) => {
+  const [hasError, setHasError] = useState(false);
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleQueryChange = useCallback((value: string) => {
+    setSearchQuery(value);
+
+    if (value) {
+      setHasError(false);
+    }
+  }, []);
+
+  const handleAdd = useCallback(() => {
+    if (selectedMovie) {
+      onAdd(selectedMovie);
+      setSelectedMovie(null);
+      setSearchQuery('');
+    }
+  }, [selectedMovie]);
+
+  const handleSubmit = useCallback(async (event:
+  React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+
```

Comment:
Pay attention to the comments
https://github.com/mate-academy/react_movies-list-fetch-movies/pull/991#discussion_r1193763045

Reply:
I cant do it. Because tests want to send a request even with an empty query